build.gradle.kts

```diff
@@ -69,6 +69,7 @@ subprojects {
             // Enforce wisp-bom -- it should take priority over external BOMs.
             add("api", enforcedPlatform(project(":wisp-bom")))
             add("api", platform(project.rootProject.libs.grpcBom))
+            add("api", platform(project.rootProject.libs.aws2Bom))
```

Comment:
AFAICT this is going to add the AWS BOM to any project that uses any of the wisp libraries. That seems a bit overreaching IMO. I'd prefer to add this as a dependency in the specific wisp subprojects that use aws libraries, so that only consumers of those libraries will get the BOM. And for that matter, it should probably match the configuration (api vs implementation) that the actual dependency (aws2Regions) is. I'll put up another PR with what I think would be better.